nat-lab/tests/telio_features.py

```diff
@@ -1,31 +1,31 @@
 from dataclasses import dataclass, field
-from dataclasses_json import dataclass_json, Undefined
+from dataclasses_json import dataclass_json, Undefined, DataClassJsonMixin
 from typing import Optional, List
 
 
 @dataclass_json
 @dataclass
-class ExitDns:
+class ExitDns(DataClassJsonMixin):
```

Comment:
Can you please explain what is this change about?

Reply:
I mean explain here in the comment, not by comments in the code

Reply:
Adding this allows linter to see, that those classes have `from_json` and `to_json` methods